calabash-cucumber/scripts/.irbrc

```diff
@@ -47,3 +43,22 @@ extend Calabash::Cucumber::Operations
 def embed(x,y=nil,z=nil)
   puts "Screenshot at #{x}"
 end
+
+require "calabash-cucumber"
+
+def preferences
+  @preferences ||= Calabash::Cucumber::Preferences.new
```

Comment:
Semantically, I'm not sure `Preferences.new` makes a lot of sense here, because there's only one 'instance'. Perhaps a static method would have made more sense, like `Preferences.fetch` or `Preferences.get`.

Also, my comments from the other PR apply here: I don't think we'd be losing a lot by reading preferences from disk each time this is called rather than storing it in an ivar, just for greater guarantee of consistent state.